### PR TITLE
added Fispact-II output file reader

### PR DESCRIPTION
**Pull request by py1sl**
Its unlikely to be  elegant or the most efficient but it seems to work although there are bound to be some cases ive not encountered
 ideas for improvements very welcome

### CODE REVIEW

File: pyne/fispact.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+def read_fis_out(path):
+    """ parse a fispact output file
+        returns fo, a fispact output object
+    """
+
+    fo = fispact_output()
+    fo.file_name = path
+    with open(path) as f:
+        lines = f.read().splitlines()
+    f.close()
```
Comment (erelson) [inline pyne/fispact.py]: with automatically closes file handles it opens, so close() is not needed

File: pyne/fispact.py [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+data
+
+this module has methods for parsing the fispact output file,
+extracting data, processing the data
+"""
+
+import numpy as np
+
+
+class fispact_output():
```
Comment (erelson) [inline pyne/fispact.py]: class names should be CamelCase per style guide/pep8.

**Comment by makeclean** [comment]
@erelson you unofficially reviewed this, are you happy to merge.

**Comment by erelson** [comment]
I did not do a proper review, per se :-x

Looked a bit more just now. Looks sane. Has documentation and tests... I never messed with fispact, so don't know the file format.

I think some of the docstrings won't be parsed correctly, as they don't fully adhere to the numpydoc style.

**Comment by makeclean** [comment]
Thanks @py1sl this is great.

**Pull request merged**